CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ All notable changes to this project will be documented in this file based on the
 
 ### Improvements
 
+- added support for the "explain" flag of AnalyzeAPI
```

Comment:
@thePanz Could you also add the PR number behind it? We missed it in the recent changelog entries but I want to add it again as it makes reading changelogs easier as you can jump to the PR directly.